### PR TITLE
Update Korean translation

### PR DESCRIPTION
**Pull request by lntuition**
Hello :) I previously translated README.md into Korean.
Also i update Korean translation. 
Check and Merge plz. thank u :)

### CODE REVIEW

**Comment by erictleung** [comment]
@lntuition thanks for taking time to update the Korean translation! I don't know Korean to be able to review this. Please try to find someone else to review the translation and approve or improve on the translation and then I will merge it. Thanks!

**Comment by lntuition** [comment]
@erictleung Thank you for comment. I ask review to @sera0731. She is now working at Canada. So She is fluent in both Korean and English.

File: README-KO.md [hunk 1]
```diff
@@ -107,18 +115,20 @@ GitHub에서 기여할 만한 적절한 이슈를 직접적으로 가리키는 
 - [Git Guide for Beginners in Spanish](https://platzi.github.io/git-slides/#/) - 스페인어로 설명된 Git과 GitHub에 대한 완벽한 안내 슬라이드.
 - [Git Kraken](https://www.gitkraken.com/git-client) - 버전 관리를 위한 비주얼, 크로스 플랫폼, 대화형 'git' 데스크톱 어플리케이션
 - [Git Tips](https://github.com/git-tips/tips) - 가장 일반적으로 사용되는 git 팁과 트릭들의 모음집
+- [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) - 자주 커밋하고, 완벽함은 나중으로, 한번만 게시하라 : Git Best Practices
```
Comment (sera0731) [inline README-KO.md]: - [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) - 커밋은 자주, 완벽함은 나중으로, 게시는 한번만. : Git Best Practices

File: README-KO.md [hunk 2]
```diff
@@ -107,18 +115,20 @@ GitHub에서 기여할 만한 적절한 이슈를 직접적으로 가리키는 
...
 - [Open Sources 2.0 : The Continuing Evolution](https://archive.org/details/opensources2.000diborich) (2005) - Open Sources 2.0은 1999년의 책인 오픈 소스 : 혁명의 목소리에서 발전된 진화적인 그림을 이어 그리는 오늘날의 기술 리더들의 통찰력과 생각을 자극하는 에세이들의 모음집입니다.
+- [The Architecture of Open Source Applications](http://www.aosabook.org/en/git.html) - 분산 워크플로우를 가능하게 하기 위해서 Git의 다양한 측면이 어떻게 작동하는지, 그리고 다른 버전 관리 시스템(VCSs)들과 어떻게 다른지 보여줍니다.
 
 ## 오픈 소스 기여 시작하기
 - [Up For Grabs](http://up-for-grabs.net/#/) - 초보자들에게 친숙한 이슈를 포함하는 프로젝트들
 - [First Timers Only](http://www.firsttimersonly.com/) - "first-timers-only" 로 라벨링된 버그 리스트
-- [First Contributions](https://firstcontributions.github.io/) - 오분 안에 오픈 소스에 처음으로 기여하세요. 초보자가 기여를 시작하는 것을 도와주는 도구와 튜토리얼이 있습니다.
-- [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - 오픈 소스 기여를 권장하는 프로그램. 10월에 최소 5번의 pull request를 통해 티셔츠나 스티커 같은 선물을 받으세요.
+- [First Contributions](https://firstcontributions.github.io/) - 오분 안에 오픈 소스에 처음으로 기여하세요. 초보자가 기여를 시작하는 것을 도와주는 도구와 튜토리얼이 있습니다. [Here](https://github.com/firstcontributions/first-contributions) is the GitHub source code for the site and opportunity to make a contribution to the repository itself.
```
Comment (sera0731) [inline README-KO.md]: - [First Contributions](https://firstcontributions.github.io/) - 오분 안에 오픈 소스에 처음으로 기여하세요. 초보자가 기여를 시작하는 것을 도와주는 도구와 튜토리얼이 있습니다. [여기](https://github.com/firstcontributions/first-contributions)는 사이트를 위한 GitHub 소스 코드이자 저장소 자체에 기여를 할 수 있는 기회입니다.

**Comment by lntuition** [comment]
@sera0731 I reflect your review. plz re-review. thank u :)

**Pull request merged**